tests/unit/bases/test_buildd.py

```diff
@@ -115,7 +115,10 @@ def test_setup(  # pylint: disable=too-many-arguments
         [*DEFAULT_FAKE_CMD, "apt-get", "install", "-y", "apt-utils"]
     )
     fake_process.register_subprocess(
-        [*DEFAULT_FAKE_CMD, "apt-get", "install", "-y", "fuse"]
+        [*DEFAULT_FAKE_CMD, "apt-get", "install", "-y", "sudo"]
+    )
+    fake_process.register_subprocess(
+        [*DEFAULT_FAKE_CMD, "apt-get", "install", "-y", "fuse", "udev"]
```

Comment:
when I asked a long time ago you told me udev was no longer needed, why is the reason to add it back?

Reply:
My understanding was that you questioned if it was still needed.  I removed it, tests worked so I thought OK, then removed it.  What I failed to realize was that I had lost the the buildd integration tests along the way and it wasn't part of the test coverage like I thought it was... :( I've got another PR to fix another regression wrt buildd in my queue and yet another to fix the coverage gap.